### ISSUE TITLE
Question: can i set cusom icon for custom percentage on custom module?

### ISSUE THREAD

**Issue by wildan-pratama** [Alexays/Waybar#2546]
i have custom audio module and i want when volume 0 (zero) show first icon
her my module
"custom/audio":{
        "format": "{percentage} {icon}",
        "return-type": "json",
        "interval": 1,
        "exec": "way-audio.sh",
        "format-icons": ["婢", ""],
        "on-click": "volume.sh mute",
        "on-click-right": "pavucontrol",
        "on-scroll-up": "volume.sh up",
		"on-scroll-down": "volume.sh down"
    },
![image](https://github.com/Alexays/Waybar/assets/84622086/7571e240-800c-4f75-944d-80970db9c3af)
![image](https://github.com/Alexays/Waybar/assets/84622086/3821fbd8-d69d-4c9f-8c37-9ff3781f01d9)

**Comment by Alexays** [comment]
Yes — recent Waybar versions let `format-icons` be an array of threshold objects, which is exactly what you want here (show the muted icon only at 0%):

```jsonc
"custom/audio": {
    "format": "{percentage} {icon}",
    "return-type": "json",
    "exec": "way-audio.sh",
    "format-icons": [
        { "icon": "婢", "max": 0 },
        { "icon": "",  "max": 100 }
    ]
}
```

The icon is picked as the first entry whose `max` is `>=` the current `percentage`, so `0` → 婢 and `1–100` → your normal icon. Just make sure `way-audio.sh` includes a numeric `percentage` field in its JSON.

(With a plain array like `["婢",""]` the range is split evenly, so the first icon shows for 0–49% — the threshold form above gives you the exact 0% behaviour.)

Closing, but reopen if it misbehaves. 🙏
